extensions/amp-form/0.1/amp-form.js

```diff
@@ -409,10 +409,7 @@ export class AmpForm {
       return;
     }
     // `submit` has the same trust level as the AMP Action that caused it.
-    this.submit_(invocation.trust);
-    if (this.method_ == 'GET' && !this.xhrAction_) {
-      this.form_.submit();
```

Comment:
Think very carefully about changing the order of significant control flows like this. Make sure to verify that the `method="GET" action="https://example.com/form` success and error cases still work as expected after this change to the submit action.

Reply:
Yeah that is one thing I definitely tried to be careful of. I have an example of all the flow cases this affects. I tested it manually and all of it seems fine. Havent tested an error state yet though, simply because I don't know a good Endpoint to hit to test this. Do we have one that just returns 500s or something?

Reply:
That should work. There are a couple of endpoints in `app.js` that serve `statusCode`s like `400` or `500`